src/styles/main.scss

```diff
@@ -208,7 +212,7 @@ article{
 	h4,
 	h5,
 	h6{
-		scroll-margin-top: 5rem;
+		scroll-margin-top: 10rem;
```

Comment:
I'll remove this - the top nav of the merged docs<>devhub banner was obscuring headlines, but since we're taking down that experiment, we don't need this anymore.